CHANGELOG.md

```diff
@@ -1,14 +1,20 @@
-# 5.0.0-beta.4 (2021-09-28)
```

Comment:
These beta releases should stay in the changelog

Reply:
fixed in commit: [#42782ea](https://github.com/blackbaud/skyux-tabs/pull/194/commits/42782ea6b644d974ac43478553d78c1d43806876)